WordPress/src/main/java/org/wordpress/android/ui/sitecreation/NewSiteCreationActivity.kt

```diff
@@ -89,6 +95,19 @@ class NewSiteCreationActivity : AppCompatActivity(),
                 finish()
             }
         })
+        mainViewModel.dialogActionObservable.observe(this, Observer { dialogHolder ->
+            dialogHolder?.let {
+                val supportFragmentManager = requireNotNull(supportFragmentManager) {
+                    "FragmentManager can't be null " +
```

Comment:
Nitpick: It looks like this could be just a single line without the `+`.

Reply:
👍